scripts/cherrypick.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -eu # Exit with nonzero exit code if anything fails
```

Comment:
please add a short comment here what this script is made for (like in tagging.sh) and how it roughly works